### ISSUE TITLE
RF tools do not function at all.

### ISSUE THREAD

**Issue by coyohti** [CGCookie/retopoflow#535]
Blender version: 2.79b
Retopoflow version: 2.0 beta 2 
OS: Windows 10

Issue:
When entering in Retopoflow mode no tools function at all. The cursor changes and "sticks" to the base mesh but no new geometry is generated.

Expected result: Ability to start generating new geometry immediately upon selecting tool (contours/polystrips/etc).

Possibly related: From watching Jonathan's video I expected RF Mode to completely replace/cover regular Blender windows but it does not in my case. I don't know if this is working as intended or another issue.

Screenshot: [https://imgur.com/sXcIr9x](url)

**Comment by carter2422** [comment]
Be sure you are holding CTRL while trying to create geometry. This has been changed from 1.x in 2.0. 

As for covering the UI, it only covers the viewport. You can use the maximize area button to Mac the viewport. You may also want to hide the toolbar and properties before starting RetopoFlow.
